pyrsia_node/src/network/behavior.rs

```diff
@@ -90,6 +98,30 @@ impl MyBehaviour {
     }
 }
 
+impl NetworkBehaviourEventProcess<gossipsub::GossipsubEvent> for MyBehaviour {
+    // Called when `gossipsub` produces an event.
+    fn inject_event(&mut self, message: gossipsub::GossipsubEvent) {
+        if let gossipsub::GossipsubEvent::Message {
+            propagation_source,
+            message_id,
+            message,
+        } = message
+        {
+            let msg_data: String = String::from_utf8(message.data).unwrap();
+            if msg_data.starts_with("magnet:") {
+                // Synapse RPC Integration point
+                println!("Start downloading {}", msg_data);
```

Comment:
node should not be printing to console =)

```suggestion
                info!("Start downloading {}", msg_data);
```

Reply:
This was a placeholder. But a logger statement serves as well. Changed.